server/data/localMockData/inmateDetailMock.ts

```diff
@@ -2073,6 +2073,20 @@ export const inmateDetailMock: InmateDetail = {
       expiredByFirstName: 'ADMIN&ONB',
       expiredByLastName: 'CNOMIS',
     },
+    {
+      alertId: 300,
+      alertType: 'H',
+      alertTypeDescription: 'Medical',
+      alertCode: 'HID',
+      alertCodeDescription: 'Hidden disability',
+      dateCreated: '2011-03-23',
+      dateExpires: '2011-04-25',
```

Comment:
Should remove this property as it's not expired

Reply:
Have removed